airflow/models.py

```diff
@@ -4363,6 +4364,21 @@ def __repr__(self):
             run_id=self.run_id,
             external_trigger=self.external_trigger)
 
+    def to_json(self):
```

Comment:
Please remove the dependency on flask in models.py. I don't think it makes sense to connect the model to the view this way. This is basically a hard coded url.